packages/animations/lib/src/modal.dart

```diff
@@ -45,7 +47,7 @@ typedef _ModalTransitionBuilder = Widget Function(
 /// the modal's characteristics.
```

Comment:
Specify in the API documentation for `showModal` that `FadeScaleTransitionConfiguration` will be used by default if it not specified.

Reply:
You need to add new tests to `packages/animations/test/modal.dart` ensuring that configuration is `FadeScaleTransitionConfiguration` by default. 

What I would do is call `showModal` without specifying a configuration, but making sure that the animations forwards and reverse are the ones for `FadeScaleTransitionConfiguration`. Another thing that would be good to test here is that the barrier properties for `FadeScaleTransitionConfiguration` are used as well.

You can use some of the existing tests in that test file and the tests from `packages/animations/test/fade_scale_transition_test.dart` to guide you.

Reply:
Thanks, I've added the tests: https://github.com/flutter/packages/pull/136/commits/b158029883e5c997c78f9759819a476e63ac7b7c